stairway/src/main/java/bio/terra/stairway/queue/QueueMessageReady.java

```diff
@@ -59,4 +70,12 @@ public String getFlightId() {
   public void setFlightId(String flightId) {
     this.flightId = flightId;
   }
+
+  public Map<String, String> getCallingThreadContext() {
```

Comment:
It would be nice to not have to add public APIs that are only used in tests.

For `getCallingThreadContext()`, could the test instead assert on `MDC.getCopyOfContextMap()`?

For `setCallingThreadContext()`, could the test instead call `MDC.setContextMap()` before constructing `QueueMessageReady`?

Reply:
Good points.

> For getCallingThreadContext(), could the test instead assert on MDC.getCopyOfContextMap()?
No, because the MDC written to the `QueueMessageReady` object on its construction can differ from the current value of the MDC.  I can remove the `public` modifier and annotate as visible for testing, though.

> For setCallingThreadContext(), could the test instead call MDC.setContextMap() before constructing QueueMessageReady?
Yes it could, and then I think I could remove this method.

Reply:
Update: I was able to remove the setter in full, but I did need to leave the getter as public so that serialization and deserialization would work.  This was caught by a unit test failure when I removed the public modifier, so that's nice :)

Other preexisting setters can likely be removed as well, but I'll designate that work as OOS.